Basic Rules.txt

```diff
@@ -1086,34 +1267,38 @@ of the caster or the Charisma Modifier.
+	Cloaking is available to some classes as a class feature.
  
 ===== Cloaking Modules =====
-Expend the cloaking module's "Nanite Activation Cost" to begin cloaking. At the
-ending of every turn thereafter, you have to pay a "Nanite Upkeep" cost to 
+	Expend the cloaking module's "Nanite Activation Cost" to begin cloaking. At 
+the ending of every turn thereafter, you have to pay a "Nanite Upkeep" cost to 
 continue cloaking.
+
 	Wearers need only expend Nanites for the "Nanite Upkeep" cost at the end 
 of a turn that they perform actions. Cloaking requires concentration so when you 
 are doing nothing but watching, you may cloak for free.
+
 	You can remain cloaked for a number of turns equal to ["Cloak Turns" + your
-cloaking module's "Cloak Turn Bonus"] or when you run out of Nanites, whichever occurs 
-sooner (see table below for "Cloak Turns") . If your cloak time runs out, you stop 
-cloaking and have to wait a number of actions equal to your "Charge Delay" (listed under 
-the relevant cloaking module's description) before being able to cloak again.
+cloaking module's "Cloak Turn Bonus"] or when you run out of Nanites, whichever 
+occurs sooner (see table below for "Cloak Turns") . If your cloak time runs out, 
+you stop cloaking and have to wait a number of actions equal to your "Charge 
+Delay" (listed under the relevant cloaking module's description) before being 
+able to cloak again.
 
 ===== Cloaking Detection =====
-If someone/something is looking for you, they must pass a (normally, perception) 
-check at 'Detect DC' difficulty, modified by your stealth skill. While moving, 
-the detect DC is reduced (listed as 'moving detect DC' under the relevant cloaking
-module's description). There is a tell-tale shimmer in the cloak, as the micro-foil 
-has a hard time keeping up with moving environments. Once the wearer has settled 
-down, the micro-foil is able to reproduce the surrounding environment better and 
-will stabilize. 
+	If someone/something is looking for you, they must pass a (normally, 
+perception) check at 'Detect DC' difficulty, modified by your stealth skill. 
+While moving, the detect DC is reduced (listed as 'moving detect DC' under the 
+relevant cloaking module's description). There is a tell-tale shimmer in the 
+cloak, as the micro-foil has a hard time keeping up with moving environments. 
+Once the wearer has settled down, the micro-foil is able to reproduce the 
+surrounding environment better and will stabilize.
+
 	When using a skill (accessing a terminal, fixing something, etc.) use your 
-reduced, 'moving detect DC' as you are diverting your attention. However, once you 
-have performed a combat action, the cloak fizzes out.
+reduced, 'moving detect DC' as you are diverting your attention. However, once 
+you have performed a combat action, the cloak fizzes out.
 
-	  CLOAKING DURATIONS AND LEVELING BONUSES
+== CLOAKING DURATIONS AND LEVELING BONUSES ==

```

Comment:
Blocker
This is not a part of "Cloaking Detection" and therefore should be a section, not a subsection.

Also why is it in angry all-caps?